backend/data_distro/migrations/0028_general_pdf_urls.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.1.7 on 2023-03-10 21:47
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("data_distro", "0027_null_for_auditee_auditee_contact"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="general",
+            name="pdf_urls",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(
+                    max_length=400,
```

Comment:
I suppose it's an easy change later, but do we want to set a higher max length (e.g. 1024) for the URL? Max object key lengths in S3 are 1024 characters, and I think URLs are a max of 2048 (in general)?

Reply:
Or, I might misunderstand this field. In which case, disregard.

Reply:
My proposal is the that PDF urls should be {AWS bucket}/collections/year/month/<file number>.pdf

My guess for that our PDF URLS would would be about 150 characters  (80 char for the bucket, 25 for the path, ~15 for the file name) so 400 gives a lot of future flexavility. But realistically,  if we are making URLS longer than 400 chars we are not giving them very good names (IMHO.)  It may be better to proxy them through the app to have shorter permalinks at a .gov address that don't change if we change buckets, but that would be a later improvement.

here is the PDF ticket that is from https://github.com/GSA-TTS/FAC/issues/751